.mvn/wrapper/maven-wrapper.properties

```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+wrapperVersion=3.3.2
+distributionType=only-script
```

Comment:
I think I would be more comfortable using the previous approach. A bit worried about odd proxy config on enterprise laptops.
Probably a good idea to let it bake for a while.

Reply:
I have restored the previous behaviour. i.e. using the binary for the root project, and using the "source" for the devtools.